db/migrate/20160329231848_text_search_indices.rb

```diff
@@ -0,0 +1,12 @@
+class TextSearchIndices < ActiveRecord::Migration
+  def change
+    DbTextSearch::CaseInsensitiveEq.add_index connection, :thredded_categories, :name, name: :thredded_categories_name_ci
+    # Recreate previously existing MySQL index in favour of consistent names
+    if connection.adapter_name.downcase =~ /mysql/
+      remove_index :thredded_topics, :title
+      remove_index :thredded_posts, :content
+    end
+    DbTextSearch::FullTextSearch.add_index connection, :thredded_topics, :title, name: :thredded_topics_title_fts
+    DbTextSearch::FullTextSearch.add_index connection, :thredded_posts, :content, name: :thredded_posts_content_fts
```

Comment:
pg is superior in every way, except not being the most popular.

1. The index previously didn't exist at all on pg.
2. The index did exist on MySQL, but under a different name.

So, this:

1. Removes the previous indexes from mysql.
2. Adds the indexes everywhere.